docs/src/apireference.md

```diff
@@ -182,11 +182,25 @@ delete(::ModelLike, ::Index)
 
 ### Variables
 
-Functions for adding variables. For deleting, see index types section.
+*Free variables* are created with with [`add_variable`](@ref) or
+[`add_variables`](@ref) and *constrained variables* are created with
```

Comment:
The discussion of constrained variables should be consistent with the MOI [standard form problem](http://www.juliaopt.org/MathOptInterface.jl/stable/apimanual/#Standard-form-problem-1). To do this, you can say that a constrained variable is equivalent to a `SingleVariable` or `VectorOfVariables` constrant.